multy_test/test_bitcoin_transaction.cpp

```diff
@@ -251,6 +250,67 @@ GTEST_TEST(BitcoinTransactionTest, SmokeTest_explicit_change)
     ASSERT_GE(available - out_1 - expected_total_fee, change_amount);
 }
 
+GTEST_TEST(BitcoinTransactionTest,SmokeTest_explicit_change2)
+{
+    AccountPtr account;
+    HANDLE_ERROR(
+            make_account(
+                    CURRENCY_BITCOIN,
+                    "cQeGKosJjWPn9GkB7QmvmotmBbVg1hm8UjdN6yLXEWZ5HAcRwam7",
+                    reset_sp(account)));
+    ASSERT_NE(nullptr, account);
+    EXPECT_EQ("mzqiDnETWkunRDZxjUQ34JzN1LDevh5DpU", account->get_address());
+
+    TransactionPtr transaction;
+    HANDLE_ERROR(make_transaction(account.get(), reset_sp(transaction)));
+    ASSERT_NE(nullptr, transaction);
+
+    const BigInt available("10000");
+    const BigInt out_1("9709");
+    const BigInt fee_per_byte("1");
+    {
+        Properties& source = transaction->add_source();
+        source.set_property_value("amount", available);
+        source.set_property_value("prev_tx_hash",
+                to_binary_data(from_hex("6e555ea9f989b755802ee69b3b7cf83777f1248fb2a678f148f71e674951cffa")));
+        source.set_property_value("prev_tx_out_index", 1u);
+        source.set_property_value("prev_tx_out_script_pubkey",
+                to_binary_data(from_hex("76a914d3f68b887224cabcc90a9581c7bbdace878666db88ac")));
+        source.set_property_value("private_key",
+                *account->get_private_key());
+    }
+
+    {
+        Properties& destination = transaction->add_destination();
+        destination.set_property_value("address",
+                                       "mpJDSHJcytfxp9asgo2pqihabHmmJkqJuM");
+        destination.set_property_value("amount", out_1);
+    }
+
+    Properties& change = transaction->add_destination();
+    change.set_property_value("address", "mzqiDnETWkunRDZxjUQ34JzN1LDevh5DpU");
+    change.set_property_value("is_change", 1);
+
+    {
+        Properties& fee = transaction->get_fee();
+        fee.set_property_value("amount_per_byte", fee_per_byte);
+    }
+
```

Comment:
Please check that:
1. Having less than 100 Staoshis in change would result in smaller TX.
2. Verify that fee is bigger when change is unprofitable.